SymStore targets disabled for System.Private.CoreLib

I temporarily disabled publishing windows pdbs to sym store as it is blocking arcade update for a week and we need to ingest it in order to unblock local live-live builds.

This is the PR where I put it: https://github.com/dotnet/runtime/pull/243 

and we should investigate why this is failing with error:

>   System.Private.CoreLib -> F:\workspace.2\_work\1\s\artifacts\bin\coreclr\Windows_NT.x64.Debug\IL\System.Private.CoreLib.dll
  The system cannot find the path specified.
F:\workspace.2\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\5.0.0-beta.19574.1\tools\SymStore.targets(57,5): error MSB3073: The command ""F:\workspace.2\_work\1\s\.packages\microsoft.diasymreader.pdb2pdb\1.1.0-beta2-19521-03\tools\Pdb2Pdb.exe" "F:\workspace.2\_work\1\s\artifacts\bin\coreclr\Windows_NT.x64.Debug\IL\System.Private.CoreLib.dll" /out "F:\workspace.2\_work\1\s\artifacts\SymStore\Debug\System.Private.CoreLib\netcoreapp2.1\System.Private.CoreLib.pdb" /srcsvrvar SRC_INDEX=public" exited with code 3. [F:\workspace.2\_work\1\s\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]

Once we do so it is a matter of removing the following from System.Private.CoreLib:
```xml
<PublishWindowsPdb>false</PublishWindowsPdb>
```

cc: @dotnet/runtime-infrastructure @tmat

--- a/src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj
+++ b/src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj
@@ -11,8 +11,6 @@
     <!-- Ensure a portable PDB is emitted for the project. A PDB is needed for crossgen. -->
     <DebugType>Portable</DebugType>
     <DebugSymbols>true</DebugSymbols>
-    <!-- Re-enable once issue: https://github.com/dotnet/runtime/issues/273 is fixed. -->
-    <PublishWindowsPdb>false</PublishWindowsPdb>
 
     <!-- Force System.Private.CoreLib.dll into a special IL output directory -->
     <OutputPath>$(BinDir)IL/</OutputPath>

